docs/users/step-debugging.md

```diff
@@ -1,90 +1,90 @@
 ## Step-debugging with ddev and xdebug
 
-Every ddev project is automatically configured with xdebug so that popular IDEs can do step-debugging of PHP code. It is disabled by default for performance reasons, so you'll need to enable it in your config.yaml.
+Every ddev project is automatically configured with xdebug so that popular IDEs can do step-debugging of PHP code
+without any server side configuration. Check the [configuration](#Configuration) section for details how you can
+change the xdebug configuration for your ddev project.
 
-xdebug is a server-side tool: It is installed automatically on the container and you do *not* need to install or configure it on your workstation.
+### Enable debugging in your IDE
 
-All IDEs basically work the same: They listen on a port and react when they're contacted there. IDEs other than those listed here work fine, if they listen on the default xdebug port 9003. (This was port 9000 through DDEV v1.18, changed to 9003 in v1.19+)
+#### PHPStorm
 
-**Key facts:**
-
-* Enable xdebug by running `ddev xdebug` or `ddev xdebug on` in your project directory. It will remain enabled until you start or restart the project.
-* Disable xdebug for better performance when not debugging with `ddev xdebug off`
-* `ddev xdebug status` will show current status.
-* The debug server port on the IDE must be set to port 9003 (port 9000 before v1.19), which is the default and is probably already set in most popular IDEs. (If you need to change the xdebug port due to a port conflict on your host computer, you can do it with a PHP override, explained below.)
-
-For more background on XDebug see [XDebug documentation](https://xdebug.org/docs/remote). The intention here is that one won't have to understand XDebug to do debugging.
-
-For each IDE the link to their documentation is provided, and the skeleton steps required are listed here.
+1. Toggle the Start Listen PHP Debug Connections button start listening php debug connections on the PhpStorm toolbar.
+   ![Start listening to debugging connections](images/ps_quick_start_zero_debug_stop_listening_debug_connections_toolbar.png)
+2. Set a breakpoint
+3. When you start debugging a [website](#Debug a website) or [CLI application](#Debug a CLI application) PhpStorm might
```

Comment:
These links don't show up right in markdown. 

![image](https://user-images.githubusercontent.com/112444/174677996-5b79ab1e-af1a-4024-bf2e-8a6039157fdf.png)

See https://github.com/drud/ddev/blob/d2ed0c21c89e1f8f9aabdb4c1bee98993d1c522e/docs/users/step-debugging.md